.github/ISSUE_TEMPLATE/5-website-removal-request.md

```diff
@@ -0,0 +1,65 @@
+---
+name: Website Removal Request
```

Comment:
Add emoji icon in the name 
name: ⛔ Website Removal Request

Reply:
Added